Description: disable test requiring a working kerberos setup
 For a yet unknown reason the check for the existence of such a setup fails
 with recent versions of MIT Kerberos (see RT#73293), so we just skip the tests
 unconditionally.
Author: Florian Schlichting <fschlich@ZEDAT.FU-Berlin.DE>
Forwarded: not-needed

--- a/t/inquire_cred.t
+++ b/t/inquire_cred.t
@@ -29,6 +29,8 @@
 };
 
 SKIP: {
+    skip('This test requires a working kerberos setup', 6);
+
     if ( $status->major != GSS_S_COMPLETE ) {
         diag( "\n\nNo error: acquire_cred() failed, maybe because you have to run kinit first.\n",
               "Errormessage from your GSSAPI-implementation is: \n\n" . qq{"$status"},
